src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj

```diff
@@ -72,6 +72,9 @@
     </None>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
Pretty sure that's what VS needs to add the project to let the NUnit test adapter run the tests from inside VS.